### PR TITLE
Adds a salad yaml and texture

### PR DESCRIPTION
**Pull request by BashieAgain**
## About the PR
This adds a new salad into the game, with a recipe and texture.

## Why / Balance
Simply added for more cooking recipes, as well as giving more use to pineapples. The recipe is easy to follow and isn't problematic to make with a decent botanist. Also felt like adding a "controversial" meal that isn't universally liked in-character.

## Media
Video attached.

- [X] I have added screenshots/videos to this PR showcasing its changes ingame

https://github.com/DeltaV-Station/Delta-v/assets/166570629/e8d0ffcf-a83a-4927-b97c-3a35726ffd56

### CODE REVIEW

File: Resources/Textures/DeltaV/Objects/Consumable/Food/hamandpineapplesalad.rsi/meta.json [hunk 1]
```diff
@@ -0,0 +1,17 @@
+{
+  "version": 1,
+  "license": "CC-BY-SA-3.0",
+  "copyright": "Bowl taken from tgstation and modified by Swept at https://github.com/tgstation/tgstation/commit/40d75cc340c63582fb66ce15bf75a36115f6bdaa, Salad made by Bashie",
```
Comment (DangerRevolution) [inline Resources/Textures/DeltaV/Objects/Consumable/Food/hamandpineapplesalad.rsi/meta.json]:
```suggestion
  "copyright": "Bowl taken from tgstation and modified by Swept at https://github.com/tgstation/tgstation/commit/40d75cc340c63582fb66ce15bf75a36115f6bdaa, Salad made by BashieAgain (Github)"
```

Or put your Discord name.

File: Resources/Textures/DeltaV/Objects/Consumable/Food/hamandpineapplesalad.rsi/meta.json [hunk 2]
```diff
@@ -0,0 +1,17 @@
...
+  "version": 1,
+  "license": "CC-BY-SA-3.0",
+  "copyright": "Bowl taken from tgstation and modified by Swept at https://github.com/tgstation/tgstation/commit/40d75cc340c63582fb66ce15bf75a36115f6bdaa, Salad made by BashieAgain (Github)"
+  "size": {
+    "x": 32,
+    "y": 32
+  },
+  "states": [
+    {
+        "name": "bowl"
```
Comment (IamVelcroboy) [inline Resources/Textures/DeltaV/Objects/Consumable/Food/hamandpineapplesalad.rsi/meta.json]: Regarding the errors, I think it's just being weird about the extra space here.

File: Resources/Textures/DeltaV/Objects/Consumable/Food/hamandpineapplesalad.rsi/meta.json [hunk 3]
```diff
@@ -0,0 +1,17 @@
...
+    },
+    {
+        "name": "hampineapplesalad"
```
Comment (IamVelcroboy) [inline Resources/Textures/DeltaV/Objects/Consumable/Food/hamandpineapplesalad.rsi/meta.json]: and here

**Comment by IamVelcroboy** [comment]
Coming back to this?

**Comment by TadJohnson00** [comment]
Considering the proposed nutriment and protein within the salad, the finished product constitutes a net loss of nutrition. If you can fix that then we can address the other issues.

**Comment by TadJohnson00** [comment]
Going to assume that this is abandonware; last changes made two months ago, no response after two weeks

**Pull request closed without merging**